crypto/merkle/proof_value.go

```diff
@@ -11,6 +11,10 @@ import (
 
 const ProofOpValue = "simple:v"
 
+// ErrTooManyArgs is returned when the input to [ValueOp.Run] has length
+// exceeding 1.
+var ErrTooManyArgs = errors.New("merkle: len(args) != 1")
```

Comment:
The message doesn't really match the error's name. Which one is right?

Reply:
Oh, I see you just moved it. You can just leave it like that, but it would be good to make error name and message consistent